imports/plugins/included/default-theme/client/styles/layout/footer.less

```diff
@@ -3,6 +3,7 @@
   position: relative;
   padding: 40px 20px;
   background-color: @footer-default-bg;
+  // background-color: rgb(41, 41, 116);
```

Comment:
Delete if not needed

Reply:
fixed.

Reply:
@Noblemajesty will this be used anywhere else? if not, then you haven't deleted it